src/services/graphqlQueries.ts

```diff
@@ -314,6 +314,44 @@ const linkQueryFields = `
     }
 `;
 
+const linksByStartNodeWithNodeUsageQuery = `
```

Comment:
getLinksWithNodeUsageByStartNodeQuery?

Reply:
linksWithNodeUsageByStartNodeQuery, since get prefix is used for functions in this file.

Reply:
but yes, I changed the word order